rds.tf

```diff
@@ -1,12 +1,48 @@
-resource "aws_db_instance" "default" {
-  allocated_storage     = 10
-  max_allocated_storage = 100
-  engine                = "mysql"
-  engine_version        = "5.7"
-  instance_class        = "db.t3.micro"
-  name                  = "mydb"
-  username              = "foo"
-  password              = "foobarbaz"
-  parameter_group_name  = "default.mysql5.7"
-  skip_final_snapshot   = true
-}
\ No newline at end of file
+resource "aws_db_instance" "mydb1" {
+  allocated_storage        = 256 # gigabytes
+  backup_retention_period  = 7   # in days
+  db_subnet_group_name     = var.rds_public_subnet_group
+  engine                   = "postgres"
+  engine_version           = "10"
+  identifier               = "mydb1"
+  instance_class           = "db.t3.micro"
+  multi_az                 = false
+  name                     = "mydb1"
+  #parameter_group_name     = "mydbparamgroup1" # if you have tuned it
+  password                 = var.db_password
+  port                     = 5432
+  publicly_accessible      = true
+  storage_encrypted        = true # you should always do this
+  storage_type             = "gp2"
+  username                 = var.db_username
+  vpc_security_group_ids   = ["${aws_security_group.mydb1.id}"]
+}
+
+resource "aws_vpc" "main" {
```

Comment:
vpc, subnets and other network resources are being created in main.tf, so this can be removed

Reply:
resolved